lib/src/main/java/scal/io/liger/ZipHelper.java

```diff
@@ -273,7 +273,7 @@ public static InputStream getThumbnailInputStreamForItem(@NonNull ExpansionIndex
         return null;
     }
 
-    private static ArrayList<String> expansionPaths;
```

Comment:
While fixing a race/NPE is good, the reason this was structured this was as a pretty important perf optimization:

https://github.com/n8fr8/storypath/commit/3815d9812bd7eced5a56afb900dede2485da9763

I'd rather fix the root crash or more carefully consider this refactor with a decent amount of testing to show that this isn't going to significantly impact our performance.